apollo-runtime/src/main/java/com/apollographql/apollo/http/HttpExecutionContext.java

```diff
@@ -0,0 +1,24 @@
+package com.apollographql.apollo.http;
+
+import com.apollographql.apollo.api.ExecutionContext;
+import com.apollographql.apollo.api.internal.Utils;
+import okhttp3.Response;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Http GraphQL execution context, provides access to the raw {@link okhttp3.Response} response.
+ */
+public class HttpExecutionContext implements ExecutionContext.Element {
```

Comment:
Might be worth making this `OkHttpExecutionContext` to make it clearer it's tied to the specific `okhttp` implementation